### PR TITLE
fix: Refactor snake case to camel case

### LINKED ISSUE #385: Refactor snake_case to camelCase
### Please describe the problem that your request should resolve.
In many places of the application we are using snake_case and camelCase interchangeably. Refactor snake_case to camelCase to unify things.

examples: 
image
image

### PR DESCRIPTION
**Pull request by janmichek**
## Description
fixes #385

## Demo
Whole app needs to be checked

## Checklist:

- [x] I have read and followed the [Contributing Guide](../../CONTRIBUTING.md)  

- [x] unify casing of attributes over app

### CODE REVIEW

File: src/stores/keyblockDetails.js [hunk 1]
```diff
@@ -13,20 +15,25 @@ export const useKeyblockDetailsStore = defineStore('keyblockDetails', () => {
     return rawKeyblock.value ? adaptKeyblock(rawKeyblock.value, keyblockDeltaStats.value) : null
   })
 
+  async function getIt(param) {
```
Comment (janmichek) [inline src/stores/keyblockDetails.js:18]: This is the solution taken from old explorer

**Comment by janmichek** [comment]
The diff is not precise, didn't start the branch from  develop
There are 3 affected files

src/components/KeyblockDetailsPanel.vue
src/utils/adapters.js
src/stores/keyblockDetails.js

Just wanted to quickly come up with the solution. The naming and placement of functions should be improved if we agree on further progress

Please take a look @lukeromanowicz

File: src/stores/keyblockDetails.js (hunk 1 above)
Comment (lukeromanowicz) [reply src/stores/keyblockDetails.js:18]: well... considering we need to apply it on every response, maybe we should just use Axios interceptors and apply camelcaseKeysDeep globally: https://axios-http.com/docs/interceptors. If you don't like this option, another good practice I saw and used in most of the projects is creating an "API Client" which is basically an Axios wrapper object put in the root catalog or utils. This gives a little bit more flexibility compared to a global interceptor.

Comment (janmichek) [reply src/stores/keyblockDetails.js:18]: Thank you, I guess Interceptor is what I wanted to use. Thanks for the hint, I will try to move this forward

**Comment by janmichek** [comment]
The branch was staged from this https://github.com/aeternity/aescan/pull/379
So there are some more files shown in diff.
Can be reviewed, however needs to be rebased on https://github.com/aeternity/aescan/pull/379 before merge

**Comment by janmichek** [comment]
Let's proceed to review here even tho, the branch is initiated from https://github.com/aeternity/aescan/pull/379 . In the diff there will be 3 more files.

**Review by lukeromanowicz — CHANGES_REQUESTED** [review]
The two most important files which are src/plugins/axios.js and src/stores/webSocket.js look really good. Everything else also looks reasonable except for one catch I found. That being said it's really hard to spot if anything is missing. I think it would be best to ask Charis to perform a full test of both mobile and desktop versions

File: src/stores/contractDetails.js [hunk 2]
```diff
@@ -54,35 +54,41 @@ export const useContractDetailsStore = defineStore('contractDetails', () => {
...
+
   async function fetchBalance() {
     const { data } = await axios.get(`${NODE_URL}/v3/accounts/${contractId.value.replace('ct_', 'ak_')}`)
     contractAccountBalance.value = data.balance
   }
+
   async function fetchContractEvents({ queryParameters = null }) {
     rawContractEvents.value = null
-    const defaultParameters = `/v2/contracts/logs?contract_id=${contractId.value}`
+    const defaultParameters = `/v2/contracts/logs?contractId=${contractId.value}`
```
[the remark this answers is not in the document]
Comment (janmichek) [reply src/stores/contractDetails.js]: Sure, I accidentally changed this when batch renaming. Fixed

**Review by lukeromanowicz — APPROVED** [review]
Looks good to me. Nice job! The only remaining thing I'm asking for is to test it carefully on desktop and mobile in its final version before merging :) Also, the sooner we merge it the better because the new code will in most cases cause conflicts

**Comment by janmichek** [comment]
@CharisSiarampalis The whole app needs to be tested

**Review by michele-franchi — CHANGES_REQUESTED** [review]
Good job but I get some errors in name and oracle details page:
image

image

Could you have a look at it?

**Comment by janmichek** [comment]
> Good job but I get some errors in name and oracle details page: image
> image
> 
> Could you have a look at it?

fixed

**Review by michele-franchi — APPROVED** [review]
It works now, good job.

**Review by CharisSiarampalis — APPROVED** [review]

**Pull request merged**